synfig-core/src/modules/lyr_freetype/lyr_freetype.cpp

```diff
@@ -271,8 +268,7 @@ class FaceCache {
 		return obj;
 	}
 
-private:
-	FaceCache() {}
+	FaceCache() = default;
```

Comment:
please, keep it private!

Reply:
Can you explain the reason to make it private?

Reply:
Isn't it a singleton?

Reply:
Got it. Fixed, please check.